examples/connext_dds/basic_security/py/secure_pub.py

```diff
@@ -0,0 +1,80 @@
+#
+# (c) 2023 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+# RTI grants Licensee a license to use, modify, compile, and create derivative
+# works of the Software solely for use with RTI products.  The Software is
+# provided "as is", with no warranty of any type, including any warranty for
+# fitness for any purpose. RTI is under no obligation to maintain or support
+# the Software.  RTI shall not be liable for any incidental or consequential
+# damages arising out of the use or inability to use the software.
+#
+
+import rti.connextdds as dds
+from hello import HelloWorld
+import time
+import argparse
+import pathlib
+
+
+XML_FILE_PATH = (
+    str(pathlib.Path(__file__).parent.absolute()) + "/SecureQos.xml"
+)
+
+
+def publisher_main(domain_id, sample_count, profile):
+    print(f"Running with profile {profile}")
+    provider = dds.QosProvider(XML_FILE_PATH)
+    qos = provider.participant_qos_from_profile(
+        f"SecurityExampleProfiles::{profile}"
+    )
+    participant = dds.DomainParticipant(domain_id, qos)
+
+    topic = dds.Topic(participant, "Example HelloWorld", HelloWorld)
+    writer = dds.DataWriter(participant.implicit_publisher, topic)
+
+    sample = HelloWorld()
+
+    # Increment count every time we send a sample
+    count = 0
+    while sample_count == 0 or count < sample_count:
+        sample.msg = f"Hello World {count}"
+        print(f"Writing sample, {sample}")
```

Comment:
```suggestion
        print(f"Writing HelloWorld Secure, count {sample}")
```
So that it matches the message that we log in the C example